Packs/SalesforceITAdmin/Integrations/Salesforce_IT_Admin/Salesforce_IT_Admin.py

```diff
@@ -0,0 +1,1164 @@
+
+    def assign_permission_set_license(self, data):
+        uri = URI_PREFIX + 'sobjects/PermissionSetLicenseAssign'
+        return self.http_request(
+            method='POST',
+            url_suffix=uri,
+            data=data
+        )
+
+    def get_assigned_permission_set_license(self, assignee_id):
+        uri = URI_PREFIX + f"query?q=SELECT+AssigneeId,Id+FROM+PermissionSetLicenseAssign+WHERE+" \
+                           f"AssigneeId='{encode_string_results(assignee_id)}'"
+        return self.http_request(
+            method='GET',
+            url_suffix=uri
+        )
+
+    def delete_assigned_permission_set_license(self, permission_set_assignment_license_id):
+        uri = URI_PREFIX + f'sobjects/PermissionSetLicenseAssign/' \
+                           f'{encode_string_results(permission_set_assignment_license_id)}'
+        return self.http_request(
+            method='DELETE',
+            url_suffix=uri
+        )
+
+    def assign_package_license(self, data):
+        uri = URI_PREFIX + 'sobjects/UserPackageLicense'
+        return self.http_request(
+            method='POST',
+            url_suffix=uri,
+            data=data
+        )
+
+    def get_assigned_package_license(self, user_id):
+        uri = URI_PREFIX + f"query?q=SELECT+Id,UserId+FROM+UserPackageLicense+WHERE+UserId=" \
+                           f"'{encode_string_results(user_id)}'"
+        return self.http_request(
+            method='GET',
+            url_suffix=uri
+        )
+
+    def delete_assigned_package_license(self, user_package_license_id):
+        uri = URI_PREFIX + f'sobjects/UserPackageLicense/' \
+                           f'{encode_string_results(user_package_license_id)}'
+        return self.http_request(
+            method='DELETE',
+            url_suffix=uri
+        )
+
+    def freeze_user_account(self, user_login_id, data):
+        uri = URI_PREFIX + f'sobjects/UserLogin/{encode_string_results(user_login_id)}'
+        params = {"_HttpMethod": "PATCH"}
+        return self.http_request(
+            method='POST',
+            url_suffix=uri,
+            params=params,
+            data=data
+        )
+
+    def unfreeze_user_account(self, user_login_id, data):
+        uri = URI_PREFIX + f'sobjects/UserLogin/{encode_string_results(user_login_id)}'
+        params = {"_HttpMethod": "PATCH"}
+        return self.http_request(
+            method='POST',
+            url_suffix=uri,
+            params=params,
+            data=data
+        )
+
+    def get_user_isfrozen_status(self, user_id):
+        uri = URI_PREFIX + f"query/?q=SELECT+Id,+IsFrozen+FROM+UserLogin+WHERE+" \
+                           f"UserId+='{encode_string_results(user_id)}'"
+        return self.http_request(
+            method='GET',
+            url_suffix=uri
+        )
+
+    # Builds a new user salesforce profile dict with pre-defined keys and custom mapping (for user)
+    def build_salesforce_profile_create_user(self, args, scim):
```

Comment:
can we unify this function and the update_user one?

Reply:
Done the changes